Assets/Project/Scripts/Repository/ModelRepository/StageModelRepository.cs

```diff
@@ -10,23 +11,27 @@ public class StageModelRepository : ModelRepositoryBase
         public static StageModelRepository Instance { get; } = new();
         
         readonly List<StageData> stages;
-        StageModel stageModel;
+        readonly List<StageModel> stageModels = new();
 
         public StageModelRepository()
         {
             dataName = "StageData";
             stages = LoadData();
+            foreach (var stage in stages)
+            {
+                stageModels.Add(new StageModel(stage));
+            }
         }
 
-        // Get()はModel毎に処理が変わる。
-        // 引数でint idを取ったりするものもあるかも
-        public StageModel Get()
+        public StageModel Get(int index)
```

Comment:
```suggestion
        public StageModel Get(int stageId)
```
の方が良い気がするな
1~7の数字を入れる感じ